### ISSUE TITLE
Unicode rendering Issue

### ISSUE THREAD

**Issue by zigarrre** [jaagr/polybar#330]
## Config

```
[bar/test]
width = 100%
font-0 = Symbola;0
modules-center = test

[module/test]
type = custom/text
content = 🔈
```

## Expected Behavior

Polybar should render a speaker symbol `🔈` (U+1F508).

## Current Behavior

Polybar renders the Replacement Character �.

## Additional Information

The problem is not limited to `🔈`, a lot of other characters/symbols don't render as well.

I'm running Arch and using the `polybar` AUR package. Symbola is installed (via the `ttf-symbola` package) an detected by `fc-match`. The character renders fine in every other application.

```
[zigarrre@desktop ~]% polybar -vv 
polybar 2.5.1

Features: +alsa +curl -i3 +mpd -network

Build type: RelWithDebInfo
Compiler: /usr/bin/c++
Compiler flags: -march=native -O2 -pipe -fstack-protector-strong --param=ssp-buffer-size=4 -D_FORTIFY_SOURCE=2 -Wall -Wextra -Werror -O2 -pedantic -pedantic-errors
Linker flags: -Wl,-O1,--sort-common,--as-needed,-z,relro

ALSA_SOUNDCARD              default
BSPWM_SOCKET_PATH           /tmp/bspwm_0_0-socket
BSPWM_STATUS_PREFIX         W
BUILDER_SPACE_TOKEN         %__
CONNECTION_TEST_IP          8.8.8.8
PATH_ADAPTER                /sys/class/power_supply/%adapter%
PATH_BACKLIGHT_MAX          /sys/class/backlight/%card%/max_brightness
PATH_BACKLIGHT_VAL          /sys/class/backlight/%card%/brightness
PATH_BATTERY                /sys/class/power_supply/%battery%
PATH_CPU_INFO               /proc/stat
PATH_MEMORY_INFO            /proc/meminfo
PATH_TEMPERATURE_INFO       /sys/class/thermal/thermal_zone%zone%/temp
```

**Comment by jaagr** [comment]
Hi, and thanks for your exemplary issue report.

Unfortunately, polybar only supports 3-byte utf-8 sequences. However, I am currently working on a new implementation for the font rendering which will have support for the upper sequences. Not sure when it will be ready though.

**Comment by jaagr** [comment]
This is fixed in the latest version.

**Comment by zigarrre** [comment]
Can confirm fix with version 3.0.1.

Thanks for the fast fix!
